extensions/database/src/com/google/refine/extension/database/DatabaseImportController.java

```diff
@@ -450,7 +450,11 @@ private DatabaseQueryInfo getQueryInfo(HttpServletRequest request) {
         jdbcConfig.setConnectionName(request.getParameter("connectionName"));
         jdbcConfig.setDatabaseType(request.getParameter("databaseType"));
         jdbcConfig.setDatabaseHost(request.getParameter("databaseServer"));
-        jdbcConfig.setDatabasePort(Integer.parseInt(request.getParameter("databasePort")));
+        try {
+        	jdbcConfig.setDatabasePort(Integer.parseInt(request.getParameter("databasePort")));
+        }catch(NumberFormatException nfE) {
```

Comment:
```suggestion
        } catch(NumberFormatException nfE) {
```

Reply:
Why was this marked as resolved when the change suggested by @wetneb (space between the brace and `catch`) wasn't made?
```suggestion
        } catch(NumberFormatException nfE) {
```